doc/code/qml_fourier.rst

```diff
@@ -0,0 +1,271 @@
+qml.fourier
+===========
+
+
+.. currentmodule:: pennylane.fourier
+
+.. automodapi:: pennylane.fourier
+    :no-heading:
+    :include-all-objects:
+    :no-inheritance-diagram:
+    :no-inherited-members:
+
```

Comment:
I think we have to add the `pennylane.fourier.visualize` case manually, since it's not automatically imported with the module. When I render the docs locally, only `coefficients` is shown in the list of module functions.

Reply:
Makes sense!